sources/ghostdog/src/main.rs

```diff
@@ -73,6 +74,11 @@ struct EfaPresentArgs {}
 /// Detect if Neuron devices are attached.
 struct NeuronPresentArgs {}
 
+#[derive(FromArgs, PartialEq, Debug)]
+#[argh(subcommand, name = "amd-gpu-present")]
```

Comment:
nit:
```suggestion
#[argh(subcommand, name = "amdgpu-present")]
``` 
Nearly everything else we refer to is all one word and the other commands all keep it to `thing`-present.